mk/icarus.mk

```diff
@@ -1 +1,11 @@
-TODO
+
+include mk/common.mk
+
+default: lint
+
+.PHONY: lint
+lint: $(addprefix ${OUTDIR}/,$(addsuffix .icarus_lint.txt,${TESTCASE_NAMES}))
+
+${OUTDIR}/%.icarus_lint.txt: testcases/%.sv
+	mkdir -p ${OUTDIR}
+	iverilog -tnull -g2012 -Wall -i $^ >> $@ 2>&1 || true
```

Comment:
I don't think the `|| true` is necessary in a Makefile, assuming it's there to allow the recipe to proceed even when the command to returns non-zero.
The correct way to mark a command as allowed-to-return-non-zero is to prefix the command with `-`. https://www.gnu.org/software/make/manual/html_node/Errors
In this case that will look like `-iverilog -tnull -g2012 -Wall -i $^ >> $@ 2>&1`

Reply:
Noted, changes will be made accordingly.